openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -589,7 +644,7 @@ def get_user_id_from_session(request):
         except KeyError:
             return None
 
-    # TODO move to test code, maybe rename, get rid of old Django compat stuff
+    # TODO move to test code, maybe rename
```

Comment:
**Reviewer Note:** Django compat code was removed in a past PR, so just updating the comment. Please mark resolved after reading.